packages/website/pages/docs/reference/contract-documentation/nft_bridge/Erc721Bridge.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Erc721Bridge
+---
+
+## Erc721Bridge
```

Comment:
For now, lets use gitignore to ignore all docs generated for these new contracts, then we add them later once and for all.